spec/light-client/supervisor/supervisor_001_draft.md

```diff
@@ -237,7 +237,7 @@ Replace_Primary(root-of-trust LightBlock)
         - pick a new secondary *nsec* while ensuring [LC-INV-ROOT-AGREED.1]
         - that is, we need to ensure that root-of-trust = FetchLightBlock(nsec, root-of-trust.Header.Height)
 - Expected precondition
-    - *FullNodes* is nonempty
+    - *FullNodes \ primary* is nonempty
```

Comment:
[LC-DATA-PEERS.1] says that the set of nodes is "partitioned", that is, that primary is not part of the set FullNodes. I think it is currently used consistently throughout the spec. I think we may clarify that in [LC-DATA-PEERS.1] rather than doing this change. What do you think?

Reply:
I see - so the *FullNodes* _doesn't_ include the primary OR the secondaries?

Reply:
yes. it is basically a pool of "unused" full nodes.

Reply:
Now, when you said it, I can see that [LC-DATA-PEERS.1] could be interpreted this way. I will clarify that rule then instead. Thanks.

Reply:
@josef-widder , changed in https://github.com/tendermint/spec/pull/333/commits/e3f5a503fbf24fb4cb06365170665d465d7041cd.